src/m365/spo/commands/site/site-apppermission-get.ts

```diff
@@ -57,14 +57,15 @@ class SpoSiteAppPermissionGetCommand extends GraphCommand {
       .getSpoSiteId(args)
       .then((siteId: string): Promise<SitePermission> => this.getApplicationPermission(args, siteId))
       .then((permissionObject: SitePermission) => {
-        const transposed: { appDisplayName: string; appId: string; permissionId: string }[] = [];
+        const transposed: { appDisplayName: string; appId: string; permissionId: string, roles: string[] }[] = [];
 
         permissionObject.grantedToIdentities.forEach((permissionEntity: SitePermissionIdentitySet) => {
           transposed.push(
             {
               appDisplayName: permissionEntity.application.displayName,
               appId: permissionEntity.application.id,
-              permissionId: permissionObject.id
+              permissionId: permissionObject.id,
+              roles: permissionObject.roles
```

Comment:
In text output mode, let's rewrite the roles array to a comma-separated string to make it more readable. The raw array doesn't add any value:

![image](https://user-images.githubusercontent.com/11164679/117536695-e09df600-affc-11eb-829a-88e054f0a7fd.png)